### DISCUSSION TITLE
add upload file menu

### DISCUSSION THREAD

**Discussion by sjwon0368** [phcode-dev/phoenix#436] [Ideas]
in phcode.dev, there isn't a menu to upload a flie

**Comment by abose** [comment]
@sjwon0368 Thanks for the idea.

Open a local folder is available in phoenix by clicking the open folder menu item. See below

Or is it some other workflows you are hinting at?

![open folder](https://user-images.githubusercontent.com/5336369/168718503-9eb1ed35-bf73-4486-bb13-99ca80263efd.gif)
